conan_provider.cmake

```diff
@@ -0,0 +1,652 @@
+# This file is managed by Conan, contents will be overwritten.
+# To keep your changes, remove these comment lines, but the plugin won't be able to modify your requirements
```

Comment:
This should be in the `cmake` subdirectory.
And question: why does it say that the file is managed by conan?